cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/status/StatusModule.java

```diff
@@ -130,4 +138,15 @@ private Measurement createMeasurement(TPStatsInfo tpStatsInfo) {
                 TimeUnit.MILLISECONDS, tags, fields);
     }
 
+    private Measurement createMeasurement(long repairSessions) {
+        final Map<String, String> tags = new HashMap<>(1);
+        tags.put("host", hostname);
+
+        final Map<String, String> fields = new HashMap<>(1);
+        fields.put("repairSessionsPending", Long.toString(repairSessions));
```

Comment:
repair sessions should be a value of this measurement. its a single measurement report